pytest-selenium/tests/test_attribution_status.py

```diff
@@ -0,0 +1,33 @@
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C476303")
+@markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+@markers.nondestructive
+def test_attribution_collapsed_by_default_expands_when_clicked(
+    selenium, base_url, book_slug, page_slug
+):
+
+    """
+    attribution section is initially collapsed, expands when clicked
+
+    GIVEN: A page URL in the format of {base_url}/books/{book_slug}/pages/{page_slug}
+    WHEN: The page is fully loaded
+    THEN: The attribution section is collapsed by default
+    AND The attribution section opens when clicked
```

Comment:
Also, not a consistent use of the `:` after the test section designator. should be `AND: `

Reply:
Updated